index.html

```diff
@@ -187,12 +187,13 @@
                 partial interface InputEvent {
                     readonly attribute DOMString     inputType;
                     readonly attribute DataTransfer? dataTransfer;
-                    sequence<StaticRange> getTargetRanges();
+                    sequence&lt;StaticRange&gt; getTargetRanges();
```

Comment:
Ah sorry, I see now it was likely me misunderstanding what foolip was asking for.